lib/Steps/ChooseIntegration.ts

```diff
@@ -1,30 +1,49 @@
 import type { Answers } from 'inquirer';
 import { prompt } from 'inquirer';
 
+import { readFileSync } from 'node:fs';
+import { hasPackageInstalled } from '../../src/utils/package-json';
 import {
   Args,
   DEFAULT_URL,
   getIntegrationChoices,
   Integration,
 } from '../Constants';
+import { dim, red } from '../Helper/Logging';
 import { BaseStep } from './BaseStep';
 import { Cordova } from './Integrations/Cordova';
 import { Electron } from './Integrations/Electron';
-import { hasPackageInstalled } from '../../src/utils/package-json';
-import { dim } from '../Helper/Logging';
-import { readFileSync } from 'node:fs';
 
 let projectPackage: Record<string, unknown> = {};
 
-try {
+const projectPackagePathCandidates = [
```

Comment:
Note to self: we should delay this when the step is actually run and use the promise-based `readFile`

Reply:
After an in-person discussion with @Lms24 we decided to just go for it, so it is now in a method and executing is deferred.